test/performance/alignment/edit_distance_unbanded_benchmark.cpp

```diff
@@ -61,15 +66,21 @@ void seqan3_edit_distance_dna4(benchmark::State & state)
     auto seq2 = generate_sequence<seqan3::dna4>(sequence_length, 0, 1);
     int score = 0;
 
-    using edit_traits_t = seqan3::detail::default_edit_distance_trait_type<std::add_lvalue_reference_t<decltype(seq1)>,
-                                                                           std::add_lvalue_reference_t<decltype(seq2)>,
+    using seq1_ref_t = std::add_lvalue_reference_t<decltype(seq1)>;
```

Comment:
uh I didn't know this type trait exists. good to know. Does this just add a `&` to the type?

Reply:
that and more. It also adheres to the reference collapsing rules.